src/lib/statusBar.ts

```diff
@@ -0,0 +1,40 @@
+import * as vscode from 'vscode';
+import { COMMANDS } from './constants';
+
+const { getConfig } = require('@hubspot/cli-lib');
+
+let hsStatusBar: vscode.StatusBarItem;
+
+export const updateStatusBarItems = () => {
+  console.log('updateStatusBarItems');
+
+  const config = getConfig();
+  const defaultAccount = config.defaultPortal;
+
+  if (defaultAccount) {
+    hsStatusBar.text = `$(arrow-swap) ${defaultAccount}`;
+    hsStatusBar.tooltip = `Active HubSpot Account: ${defaultAccount}`;
+    hsStatusBar.backgroundColor = undefined;
```

Comment:
I'm getting a TS error for that this property doesn't exist. We should probably move up the version that we support but I'm unsure what that should be

Reply:
Good call, I'm going to update the `@types/vscode` version.